<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QWidget" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>会员管理</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>会员ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>折扣</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>会员积分</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="buttonSearch">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSearch">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVID">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>会员ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>新建会员信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVName">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>180</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVPO">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.95</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>折扣</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVPoint">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>积分</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonShowAll">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>显示所有会员信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogin">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpdate">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上传会员信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>300</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelState">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆状态</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
